imports/plugins/included/simple-inventory/server/no-meteor/mutations/updateSimpleInventoryBulk.js

```diff
@@ -0,0 +1,147 @@
+import SimpleSchema from "simpl-schema";
+import Random from "@reactioncommerce/random";
+import Logger from "@reactioncommerce/logger";
+import { ProductConfigurationSchema, SimpleInventoryCollectionSchema } from "../simpleSchemas";
+
+const logCtx = { name: "core-inventory" };
+
+const inputSchema = new SimpleSchema({
+  productConfiguration: ProductConfigurationSchema,
+  canBackorder: {
+    type: Boolean,
+    optional: true
+  },
+  inventoryInStock: {
+    type: SimpleSchema.Integer,
+    min: 0,
+    optional: true
+  },
+  isEnabled: {
+    type: Boolean,
+    optional: true
+  },
+  lowInventoryWarningThreshold: {
+    type: SimpleSchema.Integer,
+    min: 0,
+    optional: true
+  },
+  shopId: String
+});
+
+const updateFields = [
+  "canBackorder",
+  "inventoryInStock",
+  "isEnabled",
+  "lowInventoryWarningThreshold"
+];
+
+const defaultValues = {
+  canBackorder: false,
+  inventoryInStock: 0,
+  isEnabled: false,
+  lowInventoryWarningThreshold: 0
+};
```

Comment:
Since these 3 const are the same in both files and we want to keep them the same in the future, I'd move them to a separate file and import them in both places.